lib/OpenQA/Log.pm

```diff
@@ -52,23 +51,23 @@ my %LOG_DEFAULTS = (LOG_TO_STANDARD_CHANNEL => 1, CHANNELS => []);
 #  log_debug("message");
 #  log_debug("message", channels=>'channel1')
 #  log_debug("message", channels=>'channel1', standard=>0)
-sub log_debug { _log_msg('debug', @_); }
+sub log_debug(@) { _log_msg('debug', @_); }
```

Comment:
I assume this kind of syntax works (never used it myself before).

Reply:
yes, see https://docs.mojolicious.org/perlsub#Signatures about "slurpy parameter"